src/client/components/rich-text/rich-text.vue

```diff
@@ -32,6 +27,11 @@ export default {
   padding: 1.3rem 0;
 }
 
+.rich-text ul,
+.rich-text ol {
+  margin-bottom: var(--spacing-default);
+}
```

Comment:
I think you are right that `.rich-text li`s could also use a little margin on the bottom (like we saw on MDN)